dozer-tests/src/e2e_tests/checker/client.rs

```diff
@@ -361,7 +373,8 @@ fn oapi_type_matches(oapi_type: &dozer_api::openapiv3::Type, field_type: FieldTy
                 true
             }
         }
-        (Array(array_type), FieldType::Binary | FieldType::Bson) => {
+        (Array(_), FieldType::Json) | (Object(_), FieldType::Json) => true,
```

Comment:
Remove this line because `FiledType::Json` doesn't match any `Type`

Reply:
agreed